modules/nodes-pods-vertical-autoscaler-custom-resource.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-vertical-autoscaler.adoc
+
+:_mod-docs-content-type: PROCEDURE
```

Comment:
This module doesn't really follow the procedure format. I would label this a reference module, though I could also see the case for a concept.